src/events.js

```diff
@@ -3096,8 +3095,7 @@ export const populateBoxManifestTable = (boxId, boxIdAndBagsObj, searchSpecimenI
             let currRow = currTable.insertRow(rowCount);
             if (j == 0) {
```

Comment:
Can make short code as below

`
 currRow.insertCell(0).innerHTML = '';
if (j == 0) {
                currRow.insertCell(0).innerHTML = bagList[i];
            }
          
`

Reply:
Thank you for the suggestion.